app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -340,8 +340,13 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
             lifecycleScope.launch {
                 // If we're authenticated, kick-off a sync and a device state refresh.
                 components.backgroundServices.accountManager.authenticatedAccount()?.let {
+                    val syncReason = when (isVisuallyComplete) {
```

Comment:
I've also include this change so we use `SyncReason.Startup` if we haven't visually completed loading.  Otherwise use `SyncReason.User` since it's the user's navigation that's triggering the sync.

Reply:
Nice!